src/main/java/Duke.java

```diff
@@ -1,10 +1,232 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.nio.file.*;
+import java.io.*;
```

Comment:
Could wildcard imports be avoided here, since there aren't many imports from these packages?